src/codeu/chat/common/UserInterest.java

```diff
@@ -0,0 +1,48 @@
+package codeu.chat.common;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import codeu.chat.util.*;
+
+public class UserInterest {
+
+	Uuid UserID; 
```

Comment:
Java naming conventions (see the style guides above) say that variables and functions should start with lower-case letters, and classes and constructors should start with upper-case letters. In other words, these variables should probably all start with lower-case letters.